### PR TITLE
dev-libs/qjson: 0.9.0 version bump, add USE="qt4,qt5" via multibuild

### PR DESCRIPTION
**Pull request by a17r**
@Pesa

### CODE REVIEW

File: dev-libs/qjson/qjson-0.9.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	multibuild_foreach_variant cmake-utils_src_compile
+}
+
+src_test() {
+	multibuild_foreach_variant cmake-utils_src_test
+}
+
+src_install() {
+	if use doc; then
+		cd doc
```
Comment (Pesa) [inline dev-libs/qjson/qjson-0.9.0.ebuild]: use pushd/popd

File: dev-libs/qjson/qjson-0.9.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+src_configure() {
+	local mycmakeargs=(
+		-DQT4_BUILD=$(usex qt4)
+	)
+
+	cmake-utils_src_configure
+}
+
+src_configure() {
```
Comment (Pesa) [inline dev-libs/qjson/qjson-0.9.0.ebuild]: you have two definitions of `src_configure`

**Comment by a17r** [comment]
That's actually a good question. I thought I would need it for qgis built with qt5, but it turned out that part was not ported yet. And then there's libkvkontakte but that really seems to be just a bogus dep.

**Comment by a17r** [comment]
How would you like to proceed? Should I
* make qt4 unconditional and keep qt5 in the event something may require it in the future
* make qt4 unconditional and drop qt5, but keep the multibuild lines intact for easy re-enabling
* drop qt5 altogether and just make this a regular bump

**Comment by Pesa** [comment]
I don't care... whatever you think is best :)

**Comment by a17r** [comment]
OK, I've dropped both flags for now and re-added multilib as found in 0.8.1.

**Pull request closed without merging**